src/lib/usbmux.ml

```diff
@@ -462,6 +466,7 @@ module Relay = struct
     |> Lwt.async
 
   let rec make_tunnels
+      ?(bind_host=None)
```

Comment:
When i am changing to `?bind_host`. I am getting compilation error.
`File "src/lib/usbmux.ml", line 551, characters 19-31:
Error: This expression has type string option
       but an expression was expected of type string`
As i am understand the parameter type becomes `string` not `optional string`.
Please help me to fix this. Sorry for so stupid questions. I first seen OCaml 2 days ago :)